airbyte-cdk/python/unit_tests/connector_builder/test_connector_builder_handler.py

```diff
@@ -457,7 +459,7 @@ def test_read():
     limits = TestReadLimits()
     with patch("airbyte_cdk.connector_builder.message_grouper.MessageGrouper.get_message_groups", return_value=stream_read):
         output_record = handle_connector_builder_request(
-            source, "test_read", config, ConfiguredAirbyteCatalog.parse_obj(CONFIGURED_CATALOG), limits
+            source, "test_read", config, ConfiguredAirbyteCatalog.parse_obj(CONFIGURED_CATALOG), _NO_STATE, limits
```

Comment:
Should we add a test here that actually sets some state, to make sure that is handled properly?

Reply:
good idea. Updated the input state and added an assertion https://github.com/airbytehq/airbyte/pull/37495/files#diff-f655e63774ac8a9d9b3b6f37f553a5e726643dc5da46d39ab11e879775b8a3edR477